pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -247,3 +253,54 @@ func (db *DB) DeleteTTLByID(id string) error {
 	}
 	return err
 }
+
+// AddBandwidthUsed adds bandwidth usage into database by date
+func (db *DB) AddBandwidthUsed(size int64, t time.Time) (err error) {
```

Comment:
you can just get time.Now() inside of this function rather passing it in

Reply:
actually you right. its not needed. removed the time argument.